demos/grasp_prediction_arc2017/.gitignore

```diff
@@ -0,0 +1 @@
+../grasp_fusion/.gitignore
```

Comment:
it is better to copy file, not linking.

Reply:
My initial motivation of using symbolic links was to keep catkin_virtualenv environment and anaconda environment of `grasp_prediction_arc2017` the same as `grasp_fusion`, because `grasp_prediction_arc2017` uses python functions of `grasp_fusion`:
e.g., https://github.com/start-jsk/jsk_apc/pull/2708/files#diff-45420299865428e11b66f5799edcf3cdR91
I used symbolic links to `grasp_fusion` for the files related to python environment settings:
https://github.com/start-jsk/jsk_apc/pull/2708/files#diff-04a9e9177e7241651a9d61d9fcaaddb8R1 ([.gitignore is to ignore auto-generated python files](https://github.com/start-jsk/jsk_apc/blob/129f6cc9cbff8739da9118157c215f4a683eb91e/demos/grasp_fusion/.gitignore))
https://github.com/start-jsk/jsk_apc/pull/2708/files#diff-1336ec4e3bf9bcff1f69f85b7529d241R1
https://github.com/start-jsk/jsk_apc/pull/2708/files#diff-1cd11b008f79219b37e8a082efca9731R1

However, now I know catkin_virtualenv can inherit the settings of other package:
https://github.com/locusrobotics/catkin_virtualenv#bundling-virtualenv
And probably anaconda problem can be solved by fixing install script.
So I'll fix the whole package not to use symbolic links.
Sorry for my lack of knowledge...

Reply:
Hmm, your linking diffs are all same.
my concern is that if `.gitignore` in `grasp_fusion` is changed, it will affect `grasp_prediction_arc2017`.
But you want to keep equal these two files, which is different from my concern, so it is good to use simlink.

Reply:
Now I deleted the symbolic links